front/app/modules/commercial/project_folders/citizen/components/ProjectFolderModerationRightsReceivedNotification/index.tsx

```diff
@@ -1,29 +1,18 @@
-import React, { ReactElement } from 'react';
-import {
-  TNotificationData,
-  IProjectFolderModerationRightsReceivedNotificationData,
-} from 'services/notifications';
+import React from 'react';
+import { IProjectFolderModerationRightsReceivedNotificationData } from 'services/notifications';
 import NotificationWrapper from 'containers/Navbar/components/NotificationMenu/components/NotificationWrapper';
 import messages from './messages';
 
 import { FormattedMessage } from 'utils/cl-intl';
 import T from 'components/T';
 
 interface Props {
-  notification:
-    | TNotificationData
-    | IProjectFolderModerationRightsReceivedNotificationData;
+  notification: IProjectFolderModerationRightsReceivedNotificationData;
 }
 
-export default function ProjectFolderModerationRightsReceivedNotification({
```

Comment:
Function definitions can't start with a capital letter (tslint), so to make components we have to use const.